src/layer/form-widgets/index.md

```diff
@@ -0,0 +1,305 @@
+
+1. Right-click on a layer, select **Properties** and go to the **Attributes form** tab.
+2. In the list of **Available Widgets** select the field you want to work with (here: `value-relation-multi-select`)
+3. In the **Widget Type** tab, select **Value Relation** from the drop-down menu and set it up as follows:
+   - Select the **Layer** that contains the values (here: `value-relation-table`)
+   - **Key column** is the field that contains the values (here: `fid`)
+   - **Value column** is the field that contains the alias (description) of the value (here: `value`)
+   - Check the **Allow multiple selections** option :heavy_check_mark: if you want to have the option to select multiple values
+4. **Apply** the changes. Don't forget to save and sync your project!
+
+![QGIS value relation field form](./qgis-form-value-relation-multiselect.jpg "QGIS value relation field form")
+
+When you open the field with **Value Relation** in the <MobileAppNameShort />, you will be able to select values from the list.
+
+![Mergin Maps mobile app value relation multiple selection](./mobile-form-value-relation.jpg "Mergin Maps mobile app value relation multiple selection")
+
+## Attachment
+
+:::tip Example project available
+Basic attachment widget is set up in the public project <MerginMapsProject id="documentation/form-widgets" />.
+:::
+
+When there is a field with the **Attachment** widget type in a survey layer, the <MobileAppNameShort /> automatically sees it as a field to **capture photos**. Such field exists in a survey layer in the basic <MainPlatformName /> project (created by using *New basic QGIS project* option when [creating a project in QGIS](../../manage/create-project/)). If you create your survey layers manually, you will need to set it up.
+
+1. Right-click on a layer, select **Properties** and go to the **Attributes form** tab.
+2. In the list of **Available Widgets** select the text field you want to work with (here: `attachment`)
+3. In the **Widget Type** tab, select **Attachment** from the drop-down menu:
+   - Default path defines where images will be saved. If it is not defined, the project home folder will be used.
+   - Store the path as **Relative to project path** or **Relative to default path** depending on how you want to store the paths.
+4. **Apply** the changes. Don't forget to save and sync your project!
+
+![QGIS form photo attachment](./qgis-form-attachment-photo.jpg "QGIS form photo attachment")
+
+In the <MobileAppNameShort />, you have the option to take a picture using the camera or upload a picture from the gallery of your mobile device:
+![Mergin Maps mobile photo attachment](./mobile-form-attachment-photo.jpg "Mergin Maps mobile photo attachment")
+
+:::tip More about this topic
+There are much more settings that can make taking photos and storing them much more efficient. Learn more about this topic in [Capturing Photos](../photos/).
+:::
+
+
+## Capturing photos, cascade forms, 1-N relations,...
```

Comment:
I'd prefer removing the "capturing photos" from here as it is already mentioned in the tip above.

```suggestion
## Cascade forms, 1-N relations,...
```